src/components/Todos/Todos.section.tsx

```diff
@@ -22,10 +25,18 @@ const Todos = ({ timerCallback }: TodosProps) => {
         setOpenAddTaskForm(false);
     }
 
+    function showCompletedTodos() {
```

Comment:
I understand that we can refactor the show/hideCompletedTodos functions as toggle function, since you click the button to toggle the state.

But the show/hideAddTaskForm I believe cannot implement as toggle function as I use them in multiple components.

Reply:
Sure, No problem.

Reply:
Have added you as one of the contributor 🙏

Reply:
Thank you so much.